apps/docs/src/.vitepress/components/BrowsersList.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts" setup>
+import { data as browsersData } from "#src/.vitepress/browser-loader.data";
+
+const browsers = browsersData.browsers.filter((b) => b.coverage > 0);
+
+const popularBrowsers = ["chrome", "edge", "firefox", "safari", "samsung", "ios_saf"];
+
+const filteredBrowsers = popularBrowsers
+  .map((pb) => {
+    const b = browsers.find((b) => b.id === pb);
+    if (b)
+      return {
+        ...b,
+        minVersion: b?.versions ? Object.keys(b.versions).sort()[0] : "",
+      };
+  })
+  .filter((b) => !!b);
+</script>
+
+<template>
+  <ul class="browsersList">
+    <li v-for="{ name, minVersion, id } in filteredBrowsers" :key="name" class="browser">
+      <img
+        :src="`/assets/browsers/${id}.svg`"
+        :alt="`${name} icon`"
+        width="40px"
+        height="40px"
```

Comment:
Should we (also) set the size via CSS? Otherwise the images are not exactly 40x40px:
![image](https://github.com/user-attachments/assets/17a0eb44-793c-4196-bd1d-a913bcfc58af)